recipes/yt_astro_analysis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "f37cd0fa7c4db58b7e577adc7c42e6f7c7f9cd1fadf62db368a3b51171c6f694" %}
+
+package:
+  name: yt_astro_analysis
+  version: {{ version }}
+
+source:
+  fn: yt_astro_analysis-{{ version }}.tar.gz
+  url: http://yt-project.org/sdist/yt_astro_analysis-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py -q install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy 1.11.*
+    - cython >=0.24.0
+    - setuptools >=19.6
+    - yt
+  run:
+    - yt
+    - setuptools
+    - python
+    - {{ pin_compatible('numpy') }}
+    - h5py
+
+test:
+  imports:
+    - yt_astro_analysis
+
+about:
+  home: http://yt-project.org/
+  license: BSD 3-clause
+  summary: Astrophysics-specific functionality built on top of yt
```

Comment:
Please add:

`license_file: COPYING.txt`

(It looks like it should be included given the `MANIFEST.in`) and

`dev_url: https://github.com/yt-project/yt_astro_analysis`

Reply:
done! thanks for the review